File: JS/J_Primes_from_1_to_n.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split(/\s+/).map(Number);

function isPrime(num) {
    if (num === 1) return false;
    for (let i = 2; i * i <= num; i++) {
        if (num % i === 0) return false;
    }
    return true;
}

const num = input[0];
let primes = [];
for (let i = 2; i <= num; i++) {
    if (isPrime(i)) primes.push(i);
}
console.log(primes.join(" "));

File: Python/J_Primes_from_1_to_n.py
def main():
    num = int(input())
    for i in range(1, num + 1):
        if isPrime(i):
            print(i, end=" ")


# def isPrime(num):
#     if num == 1: return False
#     for i in range(2, num):
#         if num % i == 0: return False
#     return True


def isPrime(num):
    if num == 1: return False
    for i in range(2, int(num ** 0.5 + 1)):
        if num % i == 0: return False
    return True


if __name__ == "__main__":
    main()